packages/account/src/providers/transaction-request/transaction-request.ts

```diff
@@ -638,19 +638,4 @@ export abstract class BaseTransactionRequest implements BaseTransactionRequestLi
   toJSON() {
     return normalizeJSON(this);
   }
-
-  /**
-   * @hidden
-   *
-   * Determines whether the transaction has a predicate input.
-   *
-   * @returns Whether the transaction has a predicate input.
-   */
-  hasPredicateInput(): boolean {
```

Comment:
This was only used in tandem with `Provider.estimatePredicates` so I inlined this logic into the body of `Provider.estimatePredicates`, specifically the variable `shouldEstimatePredicates`.